src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -0,0 +1,545 @@
+---
+title: Distributed tracing for your PHP services
+tags:
+  - Agents
+  - PHP agent
+  - Features
+metaDescription: 'How to enable distributed tracing or Infinite Tracing for the APM PHP agent.'
+redirects:
+  - /docs/agents/php-agent/features/distributed-tracing-php-agent
+  - /docs/agents/php-agent/features/distributed-tracing
+  - /docs/agents/php-agent/features/distributed-tracing-php
+  - /docs/apm/agents/php-agent/features/distributed-tracing-php-agent
+---
+
+import phplogo from 'images/phplogo.png'
+
+import LogsInContexTraces from 'images/logs-in-context-traces.png'
+
+Distributed tracing allows you to see the entire journey of your requests throughout a [distributed system](/docs/distributed-tracing/concepts/introduction-distributed-tracing). The PHP agent automatically instruments with distributed tracing a number of native PHP functions, as well as some third party HTTP clients:
+
+* PHP function [`file_get_contents`](http://php.net/manual/en/function.file-get-contents.php)
+* PHP functions [`curl_exec`](http://php.net/manual/en/function.curl-exec.php) and [`curl_multi_exec`](http://php.net/manual/en/function.curl-multi-exec.php)
+* Guzzle 4, Guzzle 5, Guzzle 6
+* Drupal's `drupal_http_request function`
+* Laravel's queue jobs
+
+For the PHP agent, we offer two types of distributed tracing (for more details, see [How span sampling works](/docs/understand-dependencies/distributed-tracing/get-started/how-new-relic-distributed-tracing-works#sampling)):
+
+* Standard (head-based sampling): Before any traces arrive, we determine a set percentage of traces to accept and analyze. This gives you a solid starting point to see how tracing can help you. It is turned on by default in PHP agents 9.21.0 and higher.
+
+* Infinite Tracing (tail-based sampling): Our cloud-based service accepts all your traces and then sorts through them to find the most important. After you set up standard tracing, we recommend you add this option because it analyzes all of your traces and gives you configuration options to sample the traces that matter most to you.
+
+All the steps you need to get started with distributed tracing are right here:
+
+* [New PHP agents:](#new-agents) Steps for head- and tail-based sampling for new agent installations
+* [Older PHP agents:](#older-agents) Tracing options if you have older PHP agents
+* [Manual instrumentation:](#manual-instrumentation) Tips if automatic instrumentation doesn't work
```

Comment:
Remove this link

Reply:
I updated the mini-toc to include these:

* [Set trace detail level](#performance)
* [Manually instrument applications and services with PHP agent API](#manual)